src/pylibsshext/channel.pyx

```diff
@@ -170,9 +180,23 @@ cdef class Channel:
 
         return result
 
+    def send_eof(self):
+        rc = libssh.ssh_channel_send_eof(self._libssh_channel)
+        if rc != libssh.SSH_OK:
+            raise LibsshChannelException("Failed to ssh_channel_send_eof: [%d]" % rc)
+
+    def send_signal(self, sig):
+        rc = libssh.ssh_channel_request_send_signal(self._libssh_channel, sig.encode("utf-8"))
+        if rc != libssh.SSH_OK:
+            raise LibsshChannelException("Failed to ssh_channel_request_send_signal: [%d]" % rc)
+
     def get_channel_exit_status(self):
         return libssh.ssh_channel_get_exit_status(self._libssh_channel)
 
+    def is_eof(self):
```

Comment:
This method doesn't depend on any arguments and its name doesn't suggest it causes a side effect (it doesn't contain a verb). So it's actually a property, semantically:
```suggestion
    @property
    def is_eof(self):
```

Reply:
Could you also add a docstring for this?